src/lib/support/ErrorStr.cpp

```diff
@@ -56,23 +56,46 @@ static ErrorFormatter * sErrorFormatterList = nullptr;
  */
 DLL_EXPORT const char * ErrorStr(CHIP_ERROR err)
 {
+    char * formattedError   = sErrorStr;
+    uint16_t formattedSpace = sizeof(sErrorStr);
+
+#if CHIP_CONFIG_ERROR_CLASS && CHIP_CONFIG_ERROR_SOURCE && !CHIP_CONFIG_SHORT_ERROR_STR
+
+    const char * const file = err.GetFile();
+    if (file != nullptr)
+    {
+        (void) snprintf(formattedError, formattedSpace, "%s:%u: ", file, err.GetLine());
+        uint16_t n = static_cast<uint16_t>(strlen(formattedError));
```

Comment:
Why not use the return value of snprintf?

Reply:
Because I'm so used to typing `(void) printf()` that I forgot it returns something useful.